contracts/vaults/Claim.sol

```diff
@@ -62,14 +63,29 @@ contract Claim is Base {
         );
     }
 
+    function challengeClaim(uint256 _claimId) external onlyArbitrator {
+        require(claims[_claimId].beneficiary != address(0), "HVE10");
+        claims[_claimId].isChallenged = true;
+    }
+
     /**
     * @notice Approve a claim for a bounty submitted by a committee, and transfer bounty to hacker and committee.
-    * Called only by hats governance.
+    * callable by the  arbitrator, if isChallenged == true
+    * Callable by anyone after challengePeriod is passed and isChallenged == false
     * @param _claimId The claim ID
     */
-    function approveClaim(uint256 _claimId) external onlyOwner nonReentrant {
+    function approveClaim(uint256 _claimId, uint256 _bountyPercentage) external nonReentrant {
         Claim storage claim = claims[_claimId];
+
```

Comment:
the check for max bounty should move here from calcClaimBounty:
        require(_bountyPercentage <= bountyInfos[_pid].maxBounty, "HVE06");

Reply:
Well it is important to check here, but not just here, because here it's only relevant if the arbitrator is the one calling